README.md

```diff
@@ -63,7 +63,7 @@ go install -v ./cmd/...
 ### Start the test network
 
 ```bash
-~/go/bin/captplanet run
+~/go/bin/captplanet run --uplink.enc-key=highlydistributedridiculouslyresilient
```

Comment:
It would be great if we can avoid requiring this additional param to run capt planet. Perhaps just for captplanet we can set a default value. It's a dev tool in the end and it does it for the API key. Yet we should avoid a default value for this uplink configuration.

Can you try setting the enc key during captplanet setup? It should be done somewhere here: https://github.com/storj/storj/blob/master/cmd/captplanet/setup.go#L114